src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+/* Fonts */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+/* Style code */
+* {
+  margin: 0;
```

Comment:
don't reset your margins with * selector, it's a bad practice, use it only for box-sizing
```suggestion
```